elastic/datadog_checks/elastic/elastic.py

```diff
@@ -222,21 +223,20 @@ def _get_index_metrics(self, admin_forwarder, version, base_tags):
                 'health': idx.get('health'),
             }
 
-            # Convert the health status value
+            # Convert Elastic health to Datadog status.
             if index_data['health'] is not None:
-                status = index_data['health'].lower()
-                index_data['health'] = health_stat[status]
-                index_data['health_reverse'] = reversed_health_stat[status]
+                dd_health = ES_HEALTH_TO_DD_STATUS[index_data['health'].lower()]
+                index_data['health'] = dd_health.status
+                index_data['health_reverse'] = dd_health.reverse_status
```

Comment:
I'm not sure the original intention was to treat these as service check statuses. They seem to be metrics, you can trace their usage down to: https://github.com/DataDog/integrations-core/blob/9945a3aa65fe19f809fd7a69ee375da5311c1c28/elastic/datadog_checks/elastic/metrics.py#L585-L586

Perhaps this was before service checks existed, so that's why it would be reported as a metric?

Reply:
Technically these are indeed metrics. I don't have exact context for why we submit the values that are identical to our service checks, `git blame` isn't informative. The code's moved around too much.

Your theory sounds plausible given that this check is on the older side. 

I think referring to our service check values is warranted here despite us submitting them as metrics. The underlying concept is the same (a factor with a limited number of values each of which maps onto the health state of ES), it's just packaged to the user differently.

Reply:
Not sure that's helpful, but fwiw some other checks are reporting some service check as metrics too: https://github.com/DataDog/integrations-core/pull/659

Reply:
I think that's a good precedent! Also I remember in rabbit we have one metric that we considered submitting as service check, but then kept it as metric only. Its values were 0 and 1 :)

Reply:
Oki, I'll resolve this thread. It looks like we agree that these particular changes are ok.